napi/Cargo.toml

```diff
@@ -19,4 +19,8 @@ bindgen = "0.54"
 glob = "0.3"
 napi-build = { version = "0.1", path = "../build" }
 regex = "1.3"
-semver = "0.10"
\ No newline at end of file
+semver = "0.10"
+
+[features]
+default = ["tsfn"]
```

Comment:
I have added a `tsfn` feature flag here because it's `N-API 4` and I think it can't get compiled for lower version of Node.js.

But it seems that we could compile the addon in a "higher" version of Node.js but use it on a lower version. (Still, we should not use higher N-API if we want the addon actually work for lower version of Node.js.) Therefore I will remove this `tsfn` flag.

Reply:
Just few lines configuration should handle this situation properly. 

```rust
// napi/lib.rs
#[cfg(napi4)]
pub mod tsfn;
```

```rust
// napi/build.rs
let napi_version = Command::new("node")
  .args(&["-e", "console.log(process.config.variables.napi_build_version)"])
  .output()
  .unwrap()
  .stdout;
for version in [1..napi_version.parse().unwrap()] {
  println!("cargo:rustc-cfg=napi{}", version);
}
```

Reply:
seems like `node@10` doesn't have `napi_build_version` property in `process.config.variables`, maybe we should set `node@10.0.0 ~ node@10.16.0` version "3" and `node@10.16+` version "4" here.